frontend/src/features/admin-form/settings/SettingsService.ts

```diff
@@ -104,6 +104,13 @@ export const updateFormWebhookRetries = async (
   })
 }
 
+export const updateFormEndPage: UpdateFormFn<'endPage'> = async (
```

Comment:
let's move this out of the `settings` folder because it's going to be shared across both the Create and Settings tabs. maybe `admin-form/common`?

Reply:
Okay, created a separate `AdminFormPageService` for this